packages/dd-trace/src/instrumenter.js

```diff
@@ -112,33 +129,38 @@ class Instrumenter {
           value: true,
           configurable: true
         })
-      })
-    })
 
-    shimmer.massWrap.call(this, nodules, names, function (original, name) {
```

Comment:
The reason this was done in 2 calls was to make sure the structure is correct to abort early otherwise.

Reply:
Can you explain further? What structure?

Reply:
Meaning that all methods are actually present, before trying to patch them.